lib/Shared/Widgets/AppDrawer.dart

```diff
@@ -40,22 +52,21 @@ class _AppDrawerState extends State<AppDrawer> {
 
   @override
   Widget build(BuildContext context) {
-    final name = FirebaseAuth.instance.currentUser!.displayName ?? "Sem Email";
-    final email = FirebaseAuth.instance.currentUser!.email ?? "Sem Nome";
+    final displayName = userName;
+    final displayEmail = userEmail;
     final image = 'assets/images/woman.png';
-    final user =
-        DatabaseMethods.chatUser ?? types.User(id: "", role: types.Role.user);
-    print(user);
+    final user = types.User(
+        id: "",
+        role: types.Role
+            .user); //arrumar a classe usuário e ifs abaixo talvez sejam desnecessários
```

Comment:
Como assim?

Reply:
Ele tem varios Ifs dentro do build para verificar se é um usuário ou se é um agente(seja lá o que isso for), mas se só um usuário do aplicativo pode entrar no aplicativo acho que essas verificações são desnecessárias. Exemplo: if (user.role == types.Role.user)

Reply:
Entendi, vamos colocar que só tem user normal mesmo, pode tirar essas verificações